index.js

```diff
@@ -14,12 +14,14 @@ const hide = hash => (sheet.innerHTML = none(hash) + sheet.innerHTML);
 const show = hash =>
   (sheet.innerHTML = sheet.innerHTML.replace(none(hash), ''));
 
+const isExternalStyleSheet = key => /^(\/|https?:\/\/)/.test(key);
```

Comment:
Can we change `.test(key)` here to `.test(key.trim())` in case formatters split the url between the back ticks onto multiple lines (like mine is doing) in which case this currently fails because of the `^` predicate.